src/pagination/pagination.component.ts

```diff
@@ -80,15 +80,12 @@ export interface PaginationTranslations {
 						</option>
 					</select>
 					<svg
-						focusable="false"
-						preserveAspectRatio="xMidYMid meet"
-						style="will-change: transform;"
-						xmlns="http://www.w3.org/2000/svg"
+						[ngClass]="{'If this is here its the new chevron': true}"
```

Comment:
Probably didn't mean to leave these in here 😉

Reply:
🙃